airbyte-integrations/connectors/source-linkedin-ads/unit_tests/test_source.py

```diff
@@ -102,10 +102,8 @@ def test_should_retry_on_error(self, error_code, requests_mock, mocker):
 
     def test_custom_streams(self):
```

Comment:
Should we have a test that actually read records from a custom stream?

Reply:
added test for custom stream read records